src/engraving/libmscore/textlinebase.cpp

```diff
@@ -149,9 +149,14 @@ void TextLineBaseSegment::draw(mu::draw::Painter* painter) const
             pen.setDashPattern({ dash, gap });
         }
 
+        // or RoundJoin, or leave at the default of MiterJoin? Make it a preference/style setting?
```

Comment:
I think we don't need this comment

Reply:
Of course, it is currently there to trigger discission about what to use and whether to make it a preference setting.
I'll remove and keep as `BevelJoin`
I'll try to create a VTest for this too.

Reply:
done